### PR TITLE
Add variable descriptions in controls bucket

### PR DESCRIPTION
**Pull request by fviale**
Also
 - add gitattribute file
 - add and fix documentation links when appropriate

### CODE REVIEW

File: Controls/resources/catalog/Wait_For_Signals.xml [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+
+
+
+          fork="true">
       <description>
         <![CDATA[ A template task that sends a ready notification for all the signals specified in the variable SIGNALS, then loops until one signal among those specified is received by the job. ]]>
       </description>
       <variables>
-        <variable name="SIGNALS" value="stop, kill" inherited="false" />
+        <variable name="SIGNALS" value="stop, kill" inherited="false"  description="List of signals that this task will expect, separated by commas."  advanced="false" hidden="false"/>
```
Comment (nebilbenmabrouk) [inline Controls/resources/catalog/Wait_For_Signals.xml:25]: List of comma-separated signals expected by this task

File: Controls/resources/catalog/Wait_For_Signals_Reactive.xml [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+
+
+
+          fork="true">
       <description>
         <![CDATA[ A template task that sends a ready notification for all the signals specified in the variable SIGNALS, then waits (i.e., performs a blocking wait) until one signal among those specified is received by the job. ]]>
       </description>
       <variables>
-        <variable name="SIGNALS" value="stop, kill" inherited="false" model="PA:REGEXP(((\w|-|_)+,?\s?)+)"/>
+        <variable name="SIGNALS" value="stop, kill" inherited="false" model="PA:REGEXP(((\w|-|_)+,?\s?)+)" description="List of signals that this task will expect, separated by commas."  advanced="false" hidden="false"/>
```
Comment (nebilbenmabrouk) [inline Controls/resources/catalog/Wait_For_Signals_Reactive.xml:25]: same here

File: Controls/resources/catalog/Wait_For_Signals.xml (hunk 1 above)
Comment (fviale) [reply Controls/resources/catalog/Wait_For_Signals.xml:25]: Isn't it "Comma-separated list of signals expected by this task" ?

Comment (fviale) [reply Controls/resources/catalog/Wait_For_Signals.xml:25]: but ok, maybe your suggestion is better

**Review by aminelouati — APPROVED** [review]

File: Controls/resources/catalog/trigger_PSA_service.xml [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
-     xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"
-     name="Trigger_PSA_Service" projectName="6. PaaS templates">
+<job
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:proactive:jobdescriptor:3.13" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd"  name="Trigger_PSA_Service" projectName="6. PaaS templates" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
     <variables>
-        <variable name="SERVICE_ID" value=""/>
-        <variable name="SERVICE_VARS" value=""/>
-        <variable name="STARTING_STATE" value="RUNNING"/>
+        <variable name="SERVICE_ID" value="" model="PA:INTEGER?" description="Id of the ProActive Service Automation service that will be triggered. You have to provide an existing SERVICE_ID from the Service Activation list in the Cloud Automation portal e.g. MongoDB, MySQL, Kibana ... ." group="" advanced="false" hidden="false"/>
```
Comment (nebilbenmabrouk) [inline Controls/resources/catalog/trigger_PSA_service.xml:6]: I prefer "Users have to .. " instead of "You have to .."

**Review by nebilbenmabrouk — APPROVED** [review]

File: Controls/resources/catalog/trigger_PSA_service.xml (hunk 3 above)
Comment (aminelouati) [reply Controls/resources/catalog/trigger_PSA_service.xml:6]: Cloud Automation portal --> Service Automation portal

File: Controls/resources/catalog/trigger_PSA_service.xml [hunk 4]
```diff
@@ -1,26 +1,29 @@
...
+        <variable name="SERVICE_VARS" value="" model="" description="&lt;b&gt;&lt;i&gt;(optional)&lt;/i&gt;/b&gt; A map as a String in format [&#39;k1&#39;: &#39;v1&#39;, &#39;k2&#39;: &#39;v2&#39;, ....] that contains a set of initialized variables used to start the service. Note that only variables of the corresponding PSA service on the cloud automation side can be initialized." group="" advanced="false" hidden="false"/>
```
Comment (aminelouati) [inline Controls/resources/catalog/trigger_PSA_service.xml:7]: cloud automation -> service automation

File: Controls/resources/catalog/execute_action_PSA_service.xml [hunk 5]
```diff
@@ -1,30 +1,30 @@
...
     <variables>
-        <variable name="INSTANCE_ID" value=""/>
-        <variable name="ACTION" value=""/>
-        <variable name="ACTION_VARS" value=""/>
+        <variable name="INSTANCE_ID" value="" model="PA:INTEGER?" description="Instance id of the active service (to be retrieved from the Activated Service list in the service-automation portal)" group="" advanced="false" hidden="false"/>
+        <variable name="ACTION" value="" model="" description="Name of the action to be executed. Possible actions are listed in the Full Services View of the started service in the service-automation portal." group="" advanced="false" hidden="false"/>
+        <variable name="ACTION_VARS" value="" model="" description="(optional) A map as a String (in format &lt;i&gt;[&#39;k1&#39;: &#39;v1&#39;, &#39;k2&#39;: &#39;v2&#39;, ....]&lt;/i&gt;) that contains a set of initialized variables used to execute the action. Note that only variables of the corresponding PSA service on the cloud automation side can be initialized." group="" advanced="false" hidden="false"/>
     </variables>
     <description>
```
Comment (aminelouati) [inline Controls/resources/catalog/execute_action_PSA_service.xml:10]: cloud automation -> service automation

File: Controls/resources/catalog/submit_job_and_wait.xml [hunk 6]
```diff
@@ -1,23 +1,29 @@
...
+
+              fork="true"
+        >
             <description>
                 <![CDATA[ Submit a workflow referenced in the ProActive Catalog (or accessible by url) and wait for its termination by checking every minute if the job is terminated. ]]>
             </description>
             <variables>
-                <variable name="called_workflow" value="basic-examples/Native_Task" inherited="false" model="PA:CATALOG_OBJECT"   advanced="false" hidden="false"/>
+                <variable name="called_workflow" value="basic-examples/Native_Task" inherited="false" model="PA:CATALOG_OBJECT" description="Workflow to submit" group="" advanced="false" hidden="false"/>
             </variables>
```
Comment (aminelouati) [inline Controls/resources/catalog/submit_job_and_wait.xml:27]: Workflow to submit --> Workflow to submit from the ProActive Catalog

File: Controls/resources/catalog/submit_job_no_wait.xml [hunk 7]
```diff
@@ -1,22 +1,29 @@
...
+
+              fork="true"
+        >
             <description>
                 <![CDATA[ Submit a workflow referenced in the ProActive Catalog (or accessible by url) without waiting for its termination ]]>
             </description>
             <variables>
-                <variable name="called_workflow" value="basic-examples/Native_Task" inherited="false" model="PA:CATALOG_OBJECT"   advanced="false" hidden="false" />
+                <variable name="called_workflow" value="basic-examples/Native_Task" inherited="false" model="PA:CATALOG_OBJECT" description="Workflow to submit" group="" advanced="false" hidden="false"/>
             </variables>
```
Comment (aminelouati) [inline Controls/resources/catalog/submit_job_no_wait.xml:27]: Workflow to submit --> Workflow to submit from the ProActive Catalog

File: Controls/resources/catalog/wait_for_any.xml [hunk 8]
```diff
@@ -1,22 +1,29 @@
...
+
+
+
+
+              fork="true"
+        >
             <variables>
-                <variable name="taskNames" value="Process_1,Process_2,Process_3" inherited="false"/>
+                <variable name="taskNames" value="Process_1,Process_2,Process_3" inherited="false"  description="Task names in the current workflow that will be waited" group="" advanced="false" hidden="false"/>
             </variables>
```
Comment (aminelouati) [inline Controls/resources/catalog/wait_for_any.xml:23]: Task names in the current workflow that will be waited -> List of comma-separated task names in the current workflow to wait for one of them to finish

**Pull request merged**